main.js

```diff
@@ -14,7 +14,9 @@ const fetchCityDetails = async (link) => {
   let result = await (await fetch(link)).json();
   try {
     printCityDetails(result["_links"]["city:urban_area"].href);
-    callPolice().then(printPoliceResults);
+    let lat = result.location.latlon.latitude;
+    let lon = result.location.latlon.longitude;
+    callPolice(lat, lon).then(printPoliceResults);
```

Comment:
This looks great! Just one small thing to add:

Can we have a try, catch for each call? That way it will help us separate issues id either printCityDetails or callPolice fail a call